solr/CHANGES.txt

```diff
@@ -171,6 +171,8 @@ Bug Fixes
 
 * SOLR-16760: bin/solr package tool supports -h and -help command like other tools.  (Eric Pugh)
 
+* SOLR-14630: When querying a single collection, SolrJ targets directly a replica of the collection (and not the collection it self) to avoid useless forwarding. (Pierre Salagnac)
```

Comment:
This is an optimization but a bug, I'd say.  This is how the user will experience it.  I would also reference "_route_" and/or "shards.preference" being pertinent.
Also "it self" -> "itself".

Suggestion:
SolrJ CloudSolrClient: when searching one collection with "_route_" or "shards.preference", form the HTTP request directly to the intended core instead of the collection.  Saves a needless HTTP hop sometimes.

Reply:
That's a good point.
Moving this to the optimization list and mentioning "shards.preference"